templates/joomla/css/custom.css

```diff
@@ -17,6 +17,18 @@
 .itemid-122 .footer {
     margin-top: 0;
 }
+
+/*error page display*/
+.error.itemid-122 #content {
```

Comment:
Need to do this without the `itemid-122` class.  https://www.joomla.org/announcements/missing.html has `itemid-154`.